server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -323,6 +337,14 @@ export class MetadataExtractionProcessor {
         }
       }
 
+      if (newExif.longitude && newExif.latitude) {
+        try {
+          newExif.timeZone = tz_lookup(newExif.latitude, newExif.longitude);
+        } catch (err) {
+          console.log('Error while calculating timezone from gps coordinates', err);
+        }
+      }
+
```

Comment:
This code is unnecessary I believe. Exiftool-vendored actually does this exact process to determine the original timezone information, so we should not need to do it a second time ourselves.

see: https://github.com/photostructure/exiftool-vendored.js#dates

The extraction information is already available here (added to `tags`)

![image](https://user-images.githubusercontent.com/4334196/221886812-148fc35b-4940-4c09-8711-0b0186a49848.png)

Reply:
Oh, I'm just re-reading your original comment... and we do indeed need this for videos still. My bad :smile: I keep forgetting that we haven't migrated to use this tool for videos as well.

Reply:
Can you update the console.log to be `this.logger.error` instead?